.travis.yml

```diff
@@ -58,6 +58,7 @@ deploy:
   skip_cleanup: true
   file_glob: true
   on:
+    condition: '$ASCIIDOCTOR_CORE_VERSION = $ASCIIDOCTOR_CORE_VERSION_PUBLISH'
```

Comment:
I think you can use `branch` instead on `condition`.

```
on:
    branch: master
```

https://docs.travis-ci.com/user/deployment#Conditional-Releases-with-on%3A